### PR TITLE
e2e: fix nil pointer errors during dereference of the enable field

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@marcel-apf FYI

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -603,6 +603,10 @@ func verifyV1alpha1Conversion(v1alpha1Profile *performancev1alpha1.PerformancePr
 		return fmt.Errorf("spec RealTimeKernel field is different")
 	}
 
+	if (specRealTimeKernel.Enabled == nil) != (v1Profile.Spec.RealTimeKernel.Enabled == nil) {
```
Comment (marcel-apf) [inline functests/1_performance/performance.go:606]: Thanks for finding it!
You may want to add the same check to the verifyV2Conversion, other than that lgtm

Comment (cynepco3hahue) [reply functests/1_performance/performance.go:606]: Good catch, fixed.

**Pull request merged**